### PR TITLE
test: collect kube-system logs for debugging

### PR DESCRIPTION
**Pull request by subhamkrai**
added kube-system namespace also to collect
logs from to debug the smoke suite issue

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

**Comment by BlaineEXE** [comment]
Do you intend for this to be merged, or is this a temporary PR for testing?

**Comment by subhamkrai** [comment]
> Do you intend for this to be merged, or is this a temporary PR for testing? Feel free to merge if you like.

yes, I would like to get this merged. I don't have access to merge 😄.

I have just rebased and pushed.

File: tests/scripts/collect-logs.sh [hunk 1]
```diff
@@ -20,6 +21,7 @@ NAMESPACES=("$CLUSTER_NAMESPACE")
 if [[ "$OPERATOR_NAMESPACE" != "$CLUSTER_NAMESPACE" ]]; then
   NAMESPACES+=("$OPERATOR_NAMESPACE")
 fi
+NAMESPACES+=("$KUBE_SYSTEM_NAMESPACE")
```
Comment (galexrt) [inline tests/scripts/collect-logs.sh:24]: Let's add the namespace to the list on line 20

Comment (subhamkrai) [reply tests/scripts/collect-logs.sh]: done

File: tests/framework/clients/filesystem.go [hunk 2]
```diff
@@ -170,7 +170,7 @@ func (f *FilesystemOperation) CreateSubvolumeGroup(fsName, groupName string) err
 		return err
 	}
 
-	err = f.k8sh.WaitForStatusPhase(namespace, "CephFilesystemSubVolumeGroup", groupName, "Ready", 15*time.Second)
+	err = f.k8sh.WaitForStatusPhase(namespace, "CephFilesystemSubVolumeGroup", groupName, "Ready", 30*time.Second)
```
Comment (parth-gr) [inline tests/framework/clients/filesystem.go:173]: why increasing the time

**Review by parth-gr — APPROVED** [review]
nit

**Pull request merged**